tests/e2e/deployment/e2e_test.go

```diff
@@ -29,8 +29,7 @@ import (
 )
 
 var (
-	nodeName     string
-	nodeSelector string
+	nodeName string
```

Comment:
device related e2e also need using nodeName?

Reply:
yes， device need to bind to a specify node when created

Reply:
lgtm 
/assign @fisherxu